BinarySearchTree/commands.c

```diff
@@ -0,0 +1,195 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "commands.h"
+#include "BinarySearchTree.h"
+
+enum command
```

Comment:
```suggestion
enum Command
```
Типы именуются с заглавной